src/Reflection/ReflectionClass.php

```diff
@@ -593,11 +595,14 @@ public function getConstant(string $name): string|int|float|bool|array|null
     {
         $reflectionConstant = $this->getReflectionConstant($name);
 
-        if (! $reflectionConstant) {
+        if ($reflectionConstant === null) {
             return null;
         }
 
-        return $reflectionConstant->getValue();
+        /** @psalm-var scalar|array<scalar>|null $constantValue */
```

Comment:
Since PHP 8.1: lies 🤣

Reply:
We don't support objects https://github.com/Roave/BetterReflection/blob/5.1.x/src/NodeCompiler/CompileNodeToValue.php#L64 :)